src/esp/gfx/PbrDrawable.h

```diff
@@ -42,8 +44,107 @@ class PbrDrawable : public Drawable {
     Mn::GL::Texture2D* roughnessTexture = nullptr;
     Mn::GL::Texture2D* metallicTexture = nullptr;
 
```

Comment:
Er, wait, I thought the shader was using a combine roughness + metallic texture? Because that's what glTF uses, and that's why roughness is in the G channel and metalness in B.

Same is for clearcoat below (factor in R, roughness in G), glTF for some reason uses separate texture properties, however you can enforce / check for this packing with [hasLayerFactorRoughnessTexture()](https://doc.magnum.graphics/magnum/classMagnum_1_1Trade_1_1PbrClearCoatMaterialData.html#aaf4e07e8fd6af18831e271a8d0e67fbd).